--- src/money.1.c	2014-07-25 16:28:37.000000000 +0300
+++ src/money.3.c	2014-07-25 16:28:37.000000000 +0300
@@ -0,0 +1,22 @@
+#include <stdlib.h>
+#include "money.h"
+
+Money *money_create(int amount, char *currency)
+{
+    return NULL;
+}
+
+int money_amount(Money * m)
+{
+    return 0;
+}
+
+char *money_currency(Money * m)
+{
+    return NULL;
+}
+
+void money_free(Money * m)
+{
+    return;
+}
